plugin-location/src/main/java/com/mapbox/maps/plugin/location/LocationPluginImpl.kt

```diff
@@ -1632,14 +1632,14 @@ class LocationPluginImpl : LocationPlugin {
    */
   override fun onStyleLoading() {
     Logger.d(TAG, "onStartLoadingMap")
-    onStartLoadingMap()
   }
```

Comment:
was this an expected change?

Reply:
with the new observable API, we don't have the `MapChange.WILL_START_LOADING_MAP` event any more.
Since it is only used in the deprecated Location Plugin, we might just walk around it.

Reply:
pushed a new commit to remove the onStyleLoading callback, and wired LocationPlugin to use onStyleLoaded.